### ISSUE TITLE
Leech Function for New Cards

### ISSUE THREAD

**Issue by whalem222** [iamjustkoi/LeechToolkit#16]
I am a medical student and I have been using Anki for 6+ years. I am currently using Anki under the assumption that it's for recalling facts that have been learned through outside resources, not for cramming facts into your head through brute memorization by forced card repetition.

That said, I'm finding that when I am learning new cards, there are some facts that I just haven't seemed to pick up during my first pass over the material. I see a card eight times and still don't have any idea what it's talking about. I would love to see a leech function for _New_ cards in order to automatically isolate these unlearned facts and get them out of the way of recalling the material that I actually have learned.

For example, say there is a card about some medication that I remember seeing, but don't actually remember anything about. I would love to see a function where if I hit Again more than three or four times in a row for that card, that card be suspended and tagged with a "New Leech" tag, or something like that. This way I'm not wasting time on those cards while reviewing, but they are also nicely organized into a single place where I could look at them at the end of the day and review their content and actually learn it, rather than wasting time trying to pound the fact into my head through rote repetition of an Anki card.

Long story short, I would love to be able to apply your very versatile Leech Toolkit to New and Learning cards as well as to Mature cards.

**Comment by iamjustkoi** [comment]
Hey! 
Sure sure, adding leech functionality for new cards defs sounds like something that might be nice to have as an option.
Can't promise it happening as soon as I'd like with my schedule the way it is right now, to be honest, but will hopefully try to see if I can get something put together as soon as I can!
